src/org/labkey/targetedms/TargetedMSController.java

```diff
@@ -1175,67 +1180,20 @@ public Object execute(QCMetricOutliersForm form, BindException errors)
 
     public static class QCPlotsDataForm
     {
-        private int _metricId;
-        private boolean _includeLJ;
-        private boolean _includeMR;
-        private boolean _includeMeanCusum;
-        private boolean _includeVariableCusum;
+        @Getter @Setter private int metricId;
+        @Getter @Setter private boolean includeLJ;
+        @Getter @Setter private boolean includeMR;
+        @Getter @Setter private boolean includeMeanCusum;
+        @Getter @Setter private boolean includeVariableCusum;
         private Date _startDate;
         private Date _endDate;
-        private List<OutlierGenerator.AnnotationGroup> _selectedAnnotations = Collections.emptyList();
-        private boolean _showExcluded;
-        private boolean _showReferenceGS;
-        private boolean _showExcludedPrecursors;
-
-        public int getMetricId()
-        {
-            return _metricId;
-        }
-
-        public void setMetricId(int metricId)
-        {
-            this._metricId = metricId;
-        }
-
-        public boolean isIncludeLJ()
-        {
-            return _includeLJ;
-        }
-
-        public void setIncludeLJ(boolean includeLJ)
-        {
-            this._includeLJ = includeLJ;
-        }
-
-        public boolean isIncludeMR()
-        {
-            return _includeMR;
-        }
-
-        public void setIncludeMR(boolean includeMR)
-        {
-            this._includeMR = includeMR;
-        }
-
-        public boolean isIncludeMeanCusum()
-        {
-            return _includeMeanCusum;
-        }
-
-        public void setIncludeMeanCusum(boolean includeMeanCusum)
-        {
-            this._includeMeanCusum = includeMeanCusum;
-        }
-
-        public boolean isIncludeVariableCusum()
-        {
-            return _includeVariableCusum;
-        }
-
-        public void setIncludeVariableCusum(boolean includeVariableCusum)
-        {
-            this._includeVariableCusum = includeVariableCusum;
-        }
+        @Getter @Setter private List<OutlierGenerator.AnnotationGroup> selectedAnnotations = Collections.emptyList();
+        @Getter @Setter  private boolean showExcluded;
+        @Getter @Setter private boolean showReferenceGS;
+        @Getter @Setter private boolean showExcludedPrecursors;
+        @Getter @Setter private int trailingRuns;
```

Comment:
Should we default this to a reasonable value like 10 so that if the API call doesn't include one, we don't pass 0 through to the calculations?